### PR TITLE
Story/quad 60

### PR DESCRIPTION
**Pull request by v1kd**
QUAD-60 On Project Details page: show the number of inactivate workspaces

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/RetrieveProjectController.java [hunk 1]
```diff
@@ -181,9 +182,15 @@ public ModelAndView getProjectDetails(@PathVariable("projectid") String projecti
...
 		List<IWorkSpace> collaboratorWorkspaceList = wsManager.listActiveWorkspaceByCollaborator(projectid, userName);
 
+        List<IWorkSpace> deactiveWorkspaceList = wsManager.listDeactivatedWorkspace(projectid, userName);
+
+        int deactivatedWSSize = deactiveWorkspaceList == null ? 0 : deactiveWorkspaceList.size();
+
 		model.getModelMap().put("project", project);
 		model.getModelMap().put("workspaceList",workspaceList);
 		model.getModelMap().put("collabworkspacelist", collaboratorWorkspaceList);
+        model.getModelMap().put("deactivatedWSSize", deactivatedWSSize);
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/RetrieveProjectController.java]: please correct indentation

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ArchiveWSController.java [hunk 2]
```diff
@@ -112,8 +112,8 @@ public ModelAndView archiveWorkspace(@Validated @ModelAttribute("workspaceform")
 		
 		archiveWorkspaceList = new ArrayList<ModifyWorkspace>();
 		workspaceId = new StringBuilder();
-		
-		if(result.hasErrors())
+
+        if(result.hasErrors())
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ArchiveWSController.java]: same here

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
ok, cool. once the formatting is correct, I'll merge it.

**Comment by v1kd** [comment]
I have formatted the code. Please go ahead with the merge.

**Pull request merged**